### DISCUSSION TITLE
Current top priorities

### DISCUSSION THREAD

**Discussion by zkoppert** [super-linter/super-linter#4501] [Maintainers]
We have identified the following categories/issues as top priorities for the project right now.

- Custom configurations for linter generate a lot of questions and may not be working properly. This could use some attention. Since there are several issues for this, it might be useful to consolidate into a single issue and provide updates there. (@zkoppert to lead this one)
- ARM builds are not working for docker containers (@Hanse00 to lead this one)
- Dependencies need continued updating.
- Old issues need clean up.
- Old discussions need addressed or cleaned up
- Establish release schedule or decision matrix for when to release
- Roadmap discussion

**Comment by zkoppert** [comment]
update on these priorities:

- Custom configurations for linter generate a lot of questions and may not be working properly. This could use some attention. Since there are several issues for this, it might be useful to consolidate into a single issue and provide updates there. (@zkoppert to lead this one)
  - Not yet completed
- ARM builds are not working for docker containers (@Hanse00 to lead this one)
  - Not yet completed
- Dependencies need continued updating.
  - This is now under control and updates are occurring and releasing weekly
- Old issues need clean up.
  - Many old issues have been cleaned up and we are now down to a reasonably manageable 34 issues at the time of writing this
- Old discussions need addressed or cleaned up
  - Completed
- Establish release schedule or decision matrix for when to release
  - Release schedule is weekly on Fridays
- Roadmap discussion
  - Not yet completed

**Reply by ferrarimarco** [reply]
I also finished closing old discussions right now :)
